rules/windows/process_creation/proc_creation_win_browsers_chromium_mockbin_abuse.yml

```diff
@@ -0,0 +1,31 @@
+title: Chromium Browser Headless Execution To Mockbin Like Site
+id: 1c526788-0abe-4713-862f-b520da5e5316
+status: experimental
+description: Detects the execution of a chromium based browser process with the "headless" flag and a URL pointing to the mockbin service
```

Comment:
```suggestion
description: Detects the execution of a Chromium based browser process with the "headless" flag and a URL pointing to the mockbin.org service (which can be used to exfiltrate data).
```
??

Reply:
Yes, its used to exfiltrate data. I'll apply